src/main/java/uk/gov/digital/ho/hocs/casework/api/factory/strategies/CopyBFToSMC.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.digital.ho.hocs.casework.api.factory.strategies;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.casework.api.CaseDataService;
+import uk.gov.digital.ho.hocs.casework.api.CorrespondentService;
+import uk.gov.digital.ho.hocs.casework.api.factory.CaseCopy;
+import uk.gov.digital.ho.hocs.casework.domain.model.CaseData;
+
+@Service
+@CaseCopy(fromCaseType = "BF", toCaseType = "SMC")
+public class CopyBFToSMC extends AbstractCaseCopyStrategy implements CaseCopyStrategy {
+
+    private static final String[] DATA_CLOB_KEYS = {
+            "Channel",
+            "CompType",
+            "CaseSummary",
```

Comment:
After you merge in the latest changes from main, can you review these? The mapping I think should be to different vars... Work done on HOCS-4590 separated the var names that these map too.

Reply:
I believe the only one of these that also exists in SMC cases is the summary, so I've added a mapping for this